src/main/webapp/app/entities/ttn/add-product-modal.tsx

```diff
@@ -219,6 +243,31 @@ export class AddProductModal extends React.Component<IAddProductModalProps, IAdd
                 <option value="REMOVED_FROM_STORAGE">REMOVED_FROM_STORAGE</option>
               </AvField>
             </AvGroup>
+            <AvGroup>
+              <Label for="rooms">
+                <span>Storage room</span>
+              </Label>
+              <AvField
+                name="rooms"
+                type="select"
+                onChange={this.handleRoomChange}
+                validate={{
+                  required: {
+                    value: true,
+                    errorMessage: translate('entity.validation.required')
+                  }
+                }}
+              >
+                <option defaultChecked />
+                {rooms
+                  ? rooms.map((otherEntity, i) => (
+                      <option value={otherEntity.id} key={i}>
```

Comment:
Since `otherEntity` has `id` it's better to assign to key `id` as well instead of `index`. But of course in this case it's no so critical.